packages/mobile/src/components/lineup-tile/LineupTileTopRight.tsx

```diff
@@ -166,9 +205,26 @@ export const LineupTileTopRight = ({
           color={neutralLight4}
         />
       )}
-      <Text style={trackTileStyles.statText}>
-        {duration ? formatLineupTileDuration(duration, isPodcast) : null}
-      </Text>
+      <View style={trackTileStyles.statTextContainer}>
+        {duration && isInProgress ? (
+          <ProgressBar
+            progress={(playbackPositionInfo.playbackPosition / duration) * 100}
+            max={100}
+            style={{ root: trackTileStyles.statTextProgressBar }}
```

Comment:
ah i think if you dont mind would be great to update ProgressBar types a bit here. `styles` is the prop that provides the object of styles, where `style` refers to `styles.root`. lmk if this is a bit annoying to do here, cause i can quickly clean up so we don't forget.